cfn.yaml

```diff
@@ -38,7 +38,7 @@ Mappings:
     CODE:
       Rate: rate(10 minutes)
     PROD:
-      Rate: rate(1 hour)
+      Rate: rate(30 minutes)
```

Comment:
I imagine this would be fine, should we need to confirm this rate with Nigel?

Reply:
Already had a chat this morning, we are getting a lot of failures at the moment because the queue is very long. We'll increase the rate temporarily while the queue clears.

Reply:
The queue was cleared manually in the meantime, so I'll put this back before merging 😄